docs/en/api/POSIX-API.md

```diff
@@ -33,29 +33,29 @@ Most basic file system operations are supported.
 However, given the intrinsic characteristics of Alluxio, like its write-once/read-many-times file
 data model, the mounted file system does not have full POSIX semantics and contains some
 limitations.
-Please read the [section of limitations](#assumptions-and-limitations) for details.
+Please read the [functionalities and limitations](#functionalities-and-limitations) for details.
 
-## Requirements
+## Quick Start Example
+
+This example shows you how to mount the whole Alluxio cluster to a local folder and run operations against the folder.
+
+### Prerequisites
 
 The followings are the basic requirements running ALLUXIO POSIX API.
-Installing Alluxio using [Docker]({{ '/en/deploy/Running-Alluxio-On-Docker.html' | relativize_url}}#enable-posix-api-access)
+Installing Alluxio POSIX API using [Docker]({{ '/en/deploy/Running-Alluxio-On-Docker.html' | relativize_url}}#enable-posix-api-access)
 and [Kubernetes]({{ '/en/deploy/Running-Alluxio-On-Kubernetes.html' | relativize_url}}#posix-api)
 can further simplify the setup.
 
+- Have a running Alluxio cluster
 - Install JDK 11, or newer
     - JDK 8 has been reported to have some bugs that may crash the FUSE applications, see [issue](https://github.com/Alluxio/alluxio/issues/15015) for more details.
 - Install libfuse
     - On Linux, we support libfuse both version 2 and 3
```

Comment:
@yyongycy is it enough? Linux and MacOS? or any other platforms are you considering?

Reply:
how about arm/amd64?

Reply:
I actually not quite sure what platforms are support and what are not, but can list out the support ones

Reply:
Added the supporting operating systems, please take a look